### PR TITLE
add button and initial config

### PR DESCRIPTION
**Pull request by Akara4ok**

### CODE REVIEW

File: src/Layout/Layout.module.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@import '../../global/scss/global.scss';
```
Comment (insouciiance) [inline src/Layout/Layout.module.scss:1]: use @use instead

Comment (insouciiance) [inline src/Layout/Layout.module.scss:1]: use `use ` instead

File: src/components/Button/Button.module.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@import '../../../global/scss/global.scss';
+
+button {
+    width: 7rem;
+    border-radius: 1rem;
+    font-size: 1rem;
+    font-family: sans-serif;
```
Comment (st1lson) [inline src/components/Button/Button.module.scss:7]: move this to global.module.scss

File: src/Layout/Layout.module.scss (hunk 1 above)
Comment (insouciiance) [inline src/Layout/Layout.module.scss]: use `use` instead

File: src/Layout/Layout.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import Button from '../components/Button/Button';
+
+const Layout = ({ children }: { children: React.ReactNode }) => {
+    return (
+        <div className={classes.container}>
+            <header>
+                <h1>Bloodit</h1>
+                <Button
+                    onClick={() => {
+                        console.log('hi');
```
Comment (st1lson) [inline src/Layout/Layout.tsx]: Remove useless console.log

Comment (Akara4ok) [reply src/Layout/Layout.tsx]: fixed

**Review by st1lson — APPROVED** [review]
merge

**Review by insouciiance — APPROVED** [review]
merge

**Pull request merged**